### PR TITLE
Chapter 7: Sign up

### PR DESCRIPTION
**Pull request by rainmaize**
61114190_418934541994662_3368476759296049152_n

### CODE REVIEW

**Comment by rainmaize** [comment]
ready

**Comment by rainmaize** [comment]
ready

**Comment by rainmaize** [comment]
Ready

File: app/helpers/users_helper.rb [hunk 1]
```diff
@@ -0,0 +1,8 @@
+module UsersHelper
+  # Returns the Gravatar for the given user.
+  def gravatar_for user, size: 80
```
Comment (dothidiemthao) [inline app/helpers/users_helper.rb]: 80 settings

File: config/routes.rb [hunk 2]
```diff
@@ -1,10 +1,10 @@
...
+ root "static_pages#home"
   get  "/help",    to: "static_pages#help"
   get  "/about",   to: "static_pages#about"
   get  "/contact", to: "static_pages#contact"
   get  "/signup",  to: "users#new"
-  root "static_pages#home"
-end
+  post "/signup",  to: "users#create"
+  resources :users
+end
```
Comment (dothidiemthao) [inline config/routes.rb]: lỗi endline, thêm 1 line trống vào cuối file

**Comment by rainmaize** [comment]
ready

File: app/helpers/users_helper.rb (hunk 1 above)
Comment (rainmaize) [reply app/helpers/users_helper.rb]: fixed

File: config/routes.rb (hunk 2 above)
Comment (rainmaize) [reply config/routes.rb]: fixed

**Comment by phamvanbk11** [comment]
merged

**Pull request merged**